src/CabalHelper/Runtime/Main.hs

```diff
@@ -19,7 +19,9 @@
 #undef CH_MIN_VERSION_Cabal
 #define CH_MIN_VERSION_Cabal MIN_VERSION_Cabal
 #endif
-
```

Comment:
Sorry about the nitpick but CPP is hard enough to read already, please preserve that newline.